Fails with `Error: Cannot find module '/home/runner/work/prisma/prisma/packages/adapter-d1/helpers/build.ts'`

Let's merge once the tests and publishing passes and make a new PR for the adapter logic

--- a/packages/adapter-d1/helpers/build.ts
+++ b/packages/adapter-d1/helpers/build.ts
@@ -1,0 +1,4 @@
+import { build } from '../../../helpers/compile/build'
+import { adapterConfig } from '../../../helpers/compile/configs'
+
+void build(adapterConfig)

--- a/packages/adapter-d1/src/index.ts
+++ b/packages/adapter-d1/src/index.ts
@@ -1,0 +1,1 @@
+export const TEMP = 42

--- a/tsconfig.build.bundle.json
+++ b/tsconfig.build.bundle.json
@@ -14,6 +14,7 @@
       "@prisma/engines": ["./packages/engines/src"],
       "@prisma/instrumentation": ["./packages/instrumentation/src"],
       "@prisma/driver-adapter-utils": ["./packages/driver-adapter-utils/src"],
+      "@prisma/adapter-d1": ["./packages/adapter-d1/src"],
       "@prisma/adapter-libsql": ["./packages/adapter-libsql/src"],
       "@prisma/adapter-neon": ["./packages/adapter-neon/src"],
       "@prisma/adapter-pg": ["./packages/adapter-pg/src"],

